packages/e2e/android/src/main/java/dev/flutter/plugins/e2e/E2EPlugin.java

```diff
@@ -13,13 +15,14 @@
 import io.flutter.plugin.common.MethodChannel.Result;
 import io.flutter.plugin.common.PluginRegistry.Registrar;
 import java.util.Map;
-import java9.util.concurrent.CompletableFuture;
 
 /** E2EPlugin */
 public class E2EPlugin implements MethodCallHandler, FlutterPlugin {
   private MethodChannel methodChannel;
 
-  public static CompletableFuture<Map<String, String>> testResults = new CompletableFuture<>();
+  private static final SettableFuture<Map<String, String>> testResultsSettable =
+      SettableFuture.create();
+  public static final ListenableFuture<Map<String, String>> testResults = testResultsSettable;
```

Comment:
Can we just make this be a `Future` instead of needing to publicly export its type?

I'm a little concerned that at some point we'll have a reason to avoid using Guava for this, and we'll have to break people again. I'm also not clear on what benefits consumers get from having this be a ListenableFuture rather than just a Future.

Reply:
Got it, done